.github/workflows/ci.yml

```diff
@@ -36,11 +36,20 @@ jobs:
             target: s390x
           - runner: ubuntu-22.04
             target: ppc64le
+          - runner: ubuntu-22.04
+            target: riscv64gc-unknown-linux-gnu
     steps:
       - uses: actions/checkout@v6
       - uses: actions/setup-python@v6
         with:
           python-version: 3.x
+      
+      - name: Set up QEMU
```

Comment:
This command will not actually run because `riscv64gc-unknown-linux-gnu != riscv64` , so can we remove it?

https://github.com/overcat/fastcrc/actions/runs/20598078040/job/59198617256?pr=27